src/containers/webuild-day/Sponsors.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Container from 'components/Container';
+import Gallery from 'components/Gallery';
+import flodeskImg from 'assets/images/webuild-day/sponsors/flodesk.png';
+import xImg from 'assets/images/webuild-day/sponsors/x.png';
+import nfqImg from 'assets/images/webuild-day/sponsors/nfq.png';
+
+const listGoldSponsors = [
+  {
+    title: 'flodesk',
+    imgUrl: flodeskImg,
+    href: ''
+  }
+];
+
+const listSilverSponsors = [
+  {
+    title: 'x',
+    imgUrl: xImg,
+    href: ''
+  },
+  {
+    title: 'nfq',
+    imgUrl: nfqImg,
+    href: ''
+  }
+];
+
+const Sponsors = () => {
+  return (
+    <section className="py-16 lg:py-24 text-gray-800 text-center font-normal">
+      <Container>
+        <div className="mb-6">
+          <h1 className="text-3xl font-bold mb-10">Sponsors</h1>
+          <div className="mb-10">
+            <p className="text-2xl font-bold mb-10">Gold</p>
+            <Gallery center items={listGoldSponsors} />
+          </div>
+          <div>
+            <p className="text-2xl font-bold mb-10">Silver</p>
+            <Gallery center items={listSilverSponsors} />
+          </div>
+        </div>
```

Comment:
Look like it's a 2px border-width?
![image](https://user-images.githubusercontent.com/12707960/100835896-f123d080-34a0-11eb-826e-cf39c97776f0.png)

Design: 
![image](https://user-images.githubusercontent.com/12707960/100835974-0dc00880-34a1-11eb-9ccc-ea217354277b.png)